### PR TITLE
 [AA][Model Service] creating Service and Ingress rule for model service

### PR DESCRIPTION
**Pull request by shsethi**

### CODE REVIEW

File: deployment/terraform/cluster/local/apps/variables.tf [hunk 1]
```diff
@@ -8,7 +8,7 @@ variable "haystack_cluster_name" {
 variable "traces" {
   type = "map"
   default = {
-    enabled = true
+    enabled = false
     version = "1.0"
```
Comment (williewheeler) [inline deployment/terraform/cluster/local/apps/variables.tf]: Was this intended?

File: deployment/terraform/cluster/local/apps/variables.tf [hunk 2]
```diff
@@ -32,7 +32,7 @@ variable "traces" {
 variable "trends" {
   type = "map"
   default = {
-    enabled = true
+    enabled = false
```
Comment (williewheeler) [inline deployment/terraform/cluster/local/apps/variables.tf]: Similarly--was this intended?

File: deployment/terraform/cluster/local/apps/variables.tf [hunk 3]
```diff
@@ -206,18 +206,78 @@ variable "metrictank" {
...
+    ewma_detector_instances = 1
+    ewma_detector_cpu_request = "100m"
+    ewma_detector_cpu_limit = "1000m"
+    ewma_detector_memory_request = "250"
+    ewma_detector_memory_limit = "250"
+    ewma_detector_jvm_memory_limit = "200"
+    ewma_detector_environment_overrides = ""
+
   }
```
Comment (williewheeler) [inline deployment/terraform/cluster/local/apps/variables.tf]: It looks like you didn't merge with my previous commit, and so are now adding these obsolete configs back to the project. :)

File: deployment/terraform/cluster/local/apps/variables.tf [hunk 4]
```diff
@@ -264,7 +325,7 @@ variable "ad-manager" {
 variable "modelservice" {
   type = "map"
   default = {
-    enabled = false
+    enabled = true
     modelservice_instances = 1
```
Comment (williewheeler) [inline deployment/terraform/cluster/local/apps/variables.tf]: Intentional? My assumption is that Haystack wants the Haystack ones true and the AA ones false for now. Is that correct?

**Comment by shsethi** [comment]
Raising a new PR 
https://github.com/ExpediaDotCom/haystack/pull/492

**Pull request closed without merging**